src/sentry/integrations/vsts/webhooks.py

```diff
@@ -43,10 +45,20 @@ def check_webhook_secret(self, request, integration):
         assert integration.metadata['subscription']['secret'] == request.META['HTTP_SHARED_SECRET']
 
     def handle_updated_workitem(self, data, integration):
-        external_issue_key = data['resource']['workItemId']
-        assigned_to = data['resource']['fields'].get('System.AssignedTo')
-        status_change = data['resource']['fields'].get('System.State')
-        project = data['resourceContainers']['project']['id']
+        try:
+            external_issue_key = data['resource']['workItemId']
+            assigned_to = data['resource']['fields'].get('System.AssignedTo')
+            status_change = data['resource']['fields'].get('System.State')
+            project = data['resourceContainers']['project']['id']
```

Comment:
should you really move all of these into this block? seems like it could be good to know if suddenly more than just `fields` starts failing in case that signals an api change we've missed.